src/Tools/babylon.tools.ts

```diff
@@ -698,6 +698,12 @@
             return false;
         }
 
+        public static getClassName(obj): string {

```

Comment:
big warning here. We used to have this function. But it won't work with minified version where code is scrambled and where constructor and names can be changed